lib/web/servers.go

```diff
@@ -65,3 +65,18 @@ func (h *Handler) clusterDatabasesGet(w http.ResponseWriter, r *http.Request, p
 
 	return ui.MakeDatabases(h.auth.clusterName, types.DeduplicateDatabases(databases)), nil
 }
+
+// // clusterDesktopsGet returns a list of desktops in a form the UI can present.
```

Comment:
Double `//`

Reply:
Found a few others throughout the codebase and fixed them as well